study.md

```diff
@@ -21,18 +21,18 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+I'm still confused on how to write a function with a loop to get multiple answers back. Also writing functions in general although I understand how they work, I'm still unsure on how to use the criteria to create the outcome I'm looking for.
```

Comment:
Here are links to the three common loop statements used in JavaScript.
for: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for
while: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/while
do...while: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/do...while

If you have a specific example that you're thinking of, then feel free to ask and I can write a sample for you.